src/components/link.js

```diff
@@ -5,8 +5,8 @@ import classNames from "classnames"
 
 import styles from "./link.module.css"
 
-const Link = ({ blank, children, color, faded, internal, size, title, to }) => {
-  const className = classNames(styles.root, styles[size], styles[color], {
+const Link = ({ blank, children, faded, internal, size, title, to }) => {
```

Comment:
Why not keep the color prop, and use the `currentColor` as a default value? I think that would be best.

Reply:
But why? As this PR shows, there are no instances where we need the link to be different than the current text color. If we need it we can always add it back, but for now there's no use for such a prop.

Reply:
Was a "just in case" scenario, but alas. Let it be